docs/en/setup/service-agent/java-agent/configuration-discovery.md

```diff
@@ -25,6 +25,7 @@ Java agent supports the following dynamic configurations.
 |        Config Key         |                      Value Description                       | Value Format Example  | Required Plugin(s) |
 | :-----------------------: | :----------------------------------------------------------: | :-------------------: | :----------------: |
 | agent.sample_n_per_3_secs |          The number of sampled traces per 3 seconds          |          -1           | - |
+| agent.ignore_suffix       |          Ignore the segments if their operation names end with these suffix. multiple paths should be separated by `,`        |          `.txt,.log`         | - |
```

Comment:
The description is not correct, take a look at `Config#Agent#IGNORE_SUFFIX`. This can't affect every operation name.

> If the operation name of the first span is included in this set, this segment should be ignored.

Reply:
Updated.